src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java

```diff
@@ -66,6 +67,28 @@
 	private static final String UMLS_ROOT = "http://www.nlm.nih.gov/research/umls/";
 	private static final org.slf4j.Logger LOGGER = org.slf4j.LoggerFactory.getLogger(ValueSetsTransformerImpl.class);
 
+	
+	public MedicationRequestStatus tActStatus2MedicationRequestStatus(String medicationRequestStatusCode) {
+		switch(medicationRequestStatusCode) {
+			case "cancelled":
```

Comment:
you are missing a few here. I count 10 on the spec (page 483), but 7 here.

Reply:
Yeah the others are defaulting to UNKNOWN I believe, I couldn't figure out a mapping for them.